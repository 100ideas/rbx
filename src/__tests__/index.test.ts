import * as rbx from "..";

describe("rbx", () => {
  it("should export all components", () => {
    expect(rbx).toMatchSnapshot();
  });
});
