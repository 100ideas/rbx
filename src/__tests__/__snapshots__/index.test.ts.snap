// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rbx should export all components 1`] = `
Object {
  "Block": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "render": [Function],
  },
  "Box": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "render": [Function],
  },
  "Breadcrumb": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Item": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "a",
      },
      "propTypes": Object {
        "active": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "nav",
    },
    "propTypes": Object {
      "align": [Function],
      "separator": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Button": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Group": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "align": [Function],
        "hasAddons": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "button",
    },
    "propTypes": Object {
      "color": [Function],
      "fullwidth": [Function],
      "inverted": [Function],
      "outlined": [Function],
      "rounded": [Function],
      "selected": [Function],
      "size": [Function],
      "state": [Function],
      "static": [Function],
      "text": [Function],
    },
    "render": [Function],
  },
  "Card": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Content": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Footer": Object {
      "$$typeof": Symbol(react.forward_ref),
      "Item": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "div",
        },
        "render": [Function],
      },
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Header": Object {
      "$$typeof": Symbol(react.forward_ref),
      "Icon": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "div",
        },
        "render": [Function],
      },
      "Title": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "div",
        },
        "render": [Function],
      },
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Image": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
    },
    "render": [Function],
  },
  "Checkbox": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "input",
    },
    "render": [Function],
  },
  "Column": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Group": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "breakpoint": [Function],
        "centered": [Function],
        "desktop": [Function],
        "fullhd": [Function],
        "gapSize": [Function],
        "gapless": [Function],
        "mobile": [Function],
        "multiline": [Function],
        "tablet": [Function],
        "touch": [Function],
        "vcentered": [Function],
        "widescreen": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "desktop": [Function],
      "fullhd": [Function],
      "mobile": [Function],
      "narrow": [Function],
      "offset": [Function],
      "size": [Function],
      "tablet": [Function],
      "touch": [Function],
      "widescreen": [Function],
    },
    "render": [Function],
  },
  "Container": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "breakpoint": [Function],
      "fluid": [Function],
    },
    "render": [Function],
  },
  "Content": Object {
    "$$typeof": Symbol(react.forward_ref),
    "OrderedList": Object {
      "$$typeof": Symbol(react.forward_ref),
      "Item": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "li",
        },
        "render": [Function],
      },
      "defaultProps": Object {
        "as": "ol",
      },
      "propTypes": Object {
        "type": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "size": [Function],
    },
    "render": [Function],
  },
  "Control": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "expanded": [Function],
      "iconLeft": [Function],
      "iconRight": [Function],
      "loading": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Delete": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "a",
    },
    "propTypes": Object {
      "size": [Function],
    },
    "render": [Function],
  },
  "Dropdown": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Container": [Function],
    "Content": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Context": Object {
      "$$typeof": Symbol(react.context),
      "Consumer": Object {
        "$$typeof": Symbol(react.context),
        "_calculateChangedBits": null,
        "_context": [Circular],
      },
      "Provider": Object {
        "$$typeof": Symbol(react.provider),
        "_context": [Circular],
      },
      "_calculateChangedBits": null,
      "_currentRenderer": null,
      "_currentRenderer2": null,
      "_currentValue": Object {
        "active": false,
        "setActive": [Function],
      },
      "_currentValue2": Object {
        "active": false,
        "setActive": [Function],
      },
      "_threadCount": 0,
    },
    "Divider": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "hr",
      },
      "render": [Function],
    },
    "Item": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "a",
      },
      "propTypes": Object {
        "active": [Function],
        "onClick": [Function],
      },
      "render": [Function],
    },
    "Menu": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Trigger": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "onClick": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "active": [Function],
      "align": [Function],
      "hoverable": [Function],
      "managed": [Function],
      "up": [Function],
    },
    "render": [Function],
  },
  "Field": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Body": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Label": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "size": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "align": [Function],
      "expanded": [Function],
      "horizontal": [Function],
      "kind": [Function],
      "multiline": [Function],
      "narrow": [Function],
    },
    "render": [Function],
  },
  "File": Object {
    "$$typeof": Symbol(react.forward_ref),
    "CTA": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Icon": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "span",
      },
      "render": [Function],
    },
    "Input": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "input",
      },
      "render": [Function],
    },
    "Label": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "label",
      },
      "render": [Function],
    },
    "Name": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "span",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "align": [Function],
      "boxed": [Function],
      "color": [Function],
      "fullwidth": [Function],
      "hasName": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Footer": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "render": [Function],
  },
  "Generic": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "as": [Function],
    },
    "render": [Function],
  },
  "Heading": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "p",
    },
    "render": [Function],
  },
  "Help": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "p",
    },
    "propTypes": Object {
      "color": [Function],
    },
    "render": [Function],
  },
  "Hero": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Body": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Foot": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Head": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "section",
    },
    "propTypes": Object {
      "color": [Function],
      "gradient": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Highlight": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "p",
    },
    "render": [Function],
  },
  "Icon": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "span",
    },
    "propTypes": Object {
      "align": [Function],
      "color": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Image": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Container": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "figure",
      },
      "propTypes": Object {
        "size": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "img",
    },
    "propTypes": Object {
      "rounded": [Function],
    },
    "render": [Function],
  },
  "Input": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "input",
    },
    "propTypes": Object {
      "color": [Function],
      "readOnly": [Function],
      "rounded": [Function],
      "size": [Function],
      "state": [Function],
      "static": [Function],
      "type": [Function],
    },
    "render": [Function],
  },
  "Label": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "label",
    },
    "propTypes": Object {
      "disabled": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Level": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Item": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "align": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "nav",
    },
    "propTypes": Object {
      "breakpoint": [Function],
    },
    "render": [Function],
  },
  "List": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Item": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "a",
      },
      "propTypes": Object {
        "active": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "hoverable": [Function],
    },
    "render": [Function],
  },
  "Loader": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
      "children": false,
    },
    "render": [Function],
  },
  "Media": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Item": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "align": "content",
        "as": "div",
      },
      "propTypes": Object {
        "align": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "article",
    },
    "render": [Function],
  },
  "Menu": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Label": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "p",
      },
      "render": [Function],
    },
    "List": Object {
      "$$typeof": Symbol(react.forward_ref),
      "Item": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "a",
        },
        "propTypes": Object {
          "active": [Function],
          "menu": [Function],
        },
        "render": [Function],
      },
      "defaultProps": Object {
        "as": "ul",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "aside",
    },
    "render": [Function],
  },
  "Message": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Body": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Header": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "article",
    },
    "propTypes": Object {
      "color": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Modal": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Background": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "onClick": [Function],
      },
      "render": [Function],
    },
    "Card": Object {
      "$$typeof": Symbol(react.forward_ref),
      "Body": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "section",
        },
        "render": [Function],
      },
      "Foot": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "footer",
        },
        "render": [Function],
      },
      "Head": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "header",
        },
        "render": [Function],
      },
      "Title": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "p",
        },
        "render": [Function],
      },
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Close": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "button",
      },
      "propTypes": Object {
        "onClick": [Function],
      },
      "render": [Function],
    },
    "Container": [Function],
    "Content": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Context": Object {
      "$$typeof": Symbol(react.context),
      "Consumer": Object {
        "$$typeof": Symbol(react.context),
        "_calculateChangedBits": null,
        "_context": [Circular],
      },
      "Provider": Object {
        "$$typeof": Symbol(react.provider),
        "_context": [Circular],
      },
      "_calculateChangedBits": null,
      "_currentRenderer": null,
      "_currentRenderer2": null,
      "_currentValue": Object {
        "close": [Function],
        "closeOnBlur": false,
        "closeOnEsc": true,
      },
      "_currentValue2": Object {
        "close": [Function],
        "closeOnBlur": false,
        "closeOnEsc": true,
      },
      "_threadCount": 0,
    },
    "Portal": [Function],
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "active": [Function],
      "closeOnBlur": [Function],
      "closeOnEsc": [Function],
      "containerClassName": [Function],
      "document": [Function],
      "onClose": [Function],
    },
    "render": [Function],
  },
  "Navbar": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Brand": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Burger": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "onClick": [Function],
        "style": [Function],
      },
      "render": [Function],
    },
    "Container": [Function],
    "Context": Object {
      "$$typeof": Symbol(react.context),
      "Consumer": Object {
        "$$typeof": Symbol(react.context),
        "_calculateChangedBits": null,
        "_context": [Circular],
      },
      "Provider": Object {
        "$$typeof": Symbol(react.provider),
        "_context": [Circular],
      },
      "_calculateChangedBits": null,
      "_currentRenderer": null,
      "_currentRenderer2": null,
      "_currentValue": Object {
        "active": false,
        "setActive": [Function],
      },
      "_currentValue2": Object {
        "active": false,
        "setActive": [Function],
      },
      "_threadCount": 0,
    },
    "Divider": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Dropdown": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "span",
      },
      "propTypes": Object {
        "align": [Function],
        "boxed": [Function],
      },
      "render": [Function],
    },
    "Item": Object {
      "$$typeof": Symbol(react.forward_ref),
      "Container": [Function],
      "defaultProps": Object {
        "as": "a",
      },
      "propTypes": Object {
        "active": [Function],
        "dropdown": [Function],
        "dropdownUp": [Function],
        "hoverable": [Function],
        "managed": [Function],
        "onClick": [Function],
      },
      "render": [Function],
    },
    "Link": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "span",
      },
      "propTypes": Object {
        "arrowless": [Function],
        "onClick": [Function],
      },
      "render": [Function],
    },
    "Menu": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Segment": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "align": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "nav",
    },
    "propTypes": Object {
      "active": [Function],
      "color": [Function],
      "document": [Function],
      "fixed": [Function],
      "managed": [Function],
      "transparent": [Function],
    },
    "render": [Function],
  },
  "Notification": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "color": [Function],
    },
    "render": [Function],
  },
  "Numeric": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "p",
    },
    "render": [Function],
  },
  "Pagination": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Ellipsis": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "span",
        "children": "…",
      },
      "render": [Function],
    },
    "Link": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "a",
      },
      "propTypes": Object {
        "current": [Function],
      },
      "render": [Function],
    },
    "List": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "ul",
      },
      "render": [Function],
    },
    "Step": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "a",
      },
      "propTypes": Object {
        "direction": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "nav",
    },
    "propTypes": Object {
      "align": [Function],
      "rounded": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Panel": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Block": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "active": [Function],
      },
      "render": [Function],
    },
    "Heading": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "Icon": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "span",
      },
      "render": [Function],
    },
    "Tabs": Object {
      "$$typeof": Symbol(react.forward_ref),
      "Tab": Object {
        "$$typeof": Symbol(react.forward_ref),
        "defaultProps": Object {
          "as": "a",
        },
        "propTypes": Object {
          "active": [Function],
        },
        "render": [Function],
      },
      "defaultProps": Object {
        "as": "div",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "nav",
    },
    "render": [Function],
  },
  "Progress": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "progress",
    },
    "propTypes": Object {
      "color": [Function],
      "max": [Function],
      "size": [Function],
      "value": [Function],
    },
    "render": [Function],
  },
  "Radio": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "input",
    },
    "render": [Function],
  },
  "Section": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "section",
    },
    "propTypes": Object {
      "size": [Function],
    },
    "render": [Function],
  },
  "Select": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Container": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "div",
      },
      "propTypes": Object {
        "color": [Function],
        "fullwidth": [Function],
        "rounded": [Function],
        "size": [Function],
        "state": [Function],
      },
      "render": [Function],
    },
    "Option": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "option",
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "select",
    },
    "render": [Function],
  },
  "Table": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "table",
    },
    "propTypes": Object {
      "bordered": [Function],
      "fullwidth": [Function],
      "hoverable": [Function],
      "narrow": [Function],
      "striped": [Function],
    },
    "render": [Function],
  },
  "Tabs": undefined,
  "Tag": Object {
    "$$typeof": Symbol(react.forward_ref),
    "Group": Object {
      "$$typeof": Symbol(react.forward_ref),
      "defaultProps": Object {
        "as": "span",
      },
      "propTypes": Object {
        "gapless": [Function],
      },
      "render": [Function],
    },
    "defaultProps": Object {
      "as": "span",
    },
    "propTypes": Object {
      "color": [Function],
      "delete": [Function],
      "rounded": [Function],
      "size": [Function],
    },
    "render": [Function],
  },
  "Textarea": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "textarea",
      "rows": 4,
    },
    "propTypes": Object {
      "color": [Function],
      "fixedSize": [Function],
      "size": [Function],
      "state": [Function],
    },
    "render": [Function],
  },
  "Tile": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "div",
    },
    "propTypes": Object {
      "kind": [Function],
      "size": [Function],
      "vertical": [Function],
    },
    "render": [Function],
  },
  "Title": Object {
    "$$typeof": Symbol(react.forward_ref),
    "defaultProps": Object {
      "as": "h1",
    },
    "propTypes": Object {
      "size": [Function],
      "spaced": [Function],
      "subtitle": [Function],
    },
    "render": [Function],
  },
  "forwardRefAs": [Function],
  "genericPropTypes": Object {
    "as": [Function],
  },
  "makeRootValidatingTransform": [Function],
}
`;
